fabric-ca-client/test/FabricCAClient.js

```diff
@@ -1080,19 +1084,8 @@ describe('FabricCAClient', () => {
 			// Should be called once
 			sinon.assert.calledOnce(requestStub);
 
-			// should call with known
-			const expected = {
```

Comment:
The test says:
> should call POST with the correct method, request, and signing identity

The test for these values has been removed so it is no longer testing what it is supposed to be testing

Reply:
@bestbeforetoday Do you mean I should add back a green test case?